client/api.c

```diff
@@ -1904,6 +1904,353 @@ TDNFUpdateInfo(
     goto cleanup;
 }
 
+uint32_t
+TDNFHistoryResolve(
+    PTDNF pTdnf,
+    PTDNF_HISTORY_ARGS pHistoryArgs,
+    PTDNF_SOLVED_PKG_INFO *ppSolvedPkgInfo)
+{
+    uint32_t dwError = 0;
+    int rc;
```

Comment:
init rc will be good although below the return value is used

Reply:
Doesn't hurt to initialize, but it's never used uninitialized.